L1Trigger/L1TTrackMatch/python/L1TkMuonProducer_cfi.py

```diff
@@ -49,3 +49,7 @@
 
 
 )
+
+L1TkMuonsDynamicWindows = L1TkMuons.Clone(emtfMatchAlgoVersion='DynamicWindows')
+
+L1TkMuons = cms.Sequence(L1TkMuons * L1TkMuonsDynamicWindows)
```

Comment:
Rename this so it does not have the same name as the L1TkMuon object (eg: L1TkMuonSequence ?)